_admin_guides/rosco.md

```diff
@@ -145,14 +145,14 @@ Once you have the custom template and script deployed onto your Spinnaker cluste
 The default configurations use `Ubuntu 12.04/14.0` as the default choices for Base OS - these are configurable by adding the configurations below to your `rosco-local.yml`.   You can specify a different `templateFile` per base image which should save time from a user perspective so they don't have to specify an additional parameter.  You can specify multiple base images and visualization settings for each region you need to process bakes.
 
 > **Note** There's currently a bug in the YAML merge that if you specify less than 3 `baseImages` (listed in `rosco.yml`), Spinnaker will show `null null` in the drop down for **Base OS**.
- 
+
 
 Here's a simple version of `rosco-local.yml`. See [Full Version](https://github.com/spinnaker/rosco/blob/master/rosco-web/config/rosco.yml)
 ```
 aws:
   enabled: true
   bakeryDefaults:
-    awsAssociatePublicIpAddress: false
+    awsAssociatePublicIpAddress: true
```

Comment:
the commit message doesn't fit with this. Why did this need to be changed?

Reply:
Ahh, that was a mistake.